Add docs for clustering (#129)
AlexsLemonade/scpca-docs: docs/processing_information.md

These HVGs are then used as input to principal component analysis, and the top 50 principal components are selected.
Finally, these principal components are used to calculate the [UMAP (Uniform Manifold Approximation and Projection)](http://bioconductor.org/books/3.13/OSCA.basic/dimensionality-reduction.html#uniform-manifold-approximation-and-projection) embeddings.

> [Graph-based clustering](http://bioconductor.org/books/3.16/OSCA.basic/clustering.html#clustering-graph) is also performed, using by default the Louvain algorithm with 20 nearest neighbors, using Jaccard weighting.

Reviewer: ```suggestion
[Graph-based clustering](http://bioconductor.org/books/3.16/OSCA.basic/clustering.html#clustering-graph) is also performed, using the Louvain algorithm with 20 nearest neighbors and Jaccard weighting.
```
